content/applications/sales/point_of_sale/shop/invoice.rst

```diff
@@ -58,6 +58,23 @@ To retrieve invoices from the **POS dashboard**,
 
 .. note::
    - **Invoiced orders** can be identified by the :guilabel:`Invoiced` status in the
-     :guilabel:`Status` column;
+     :guilabel:`Status` column.
    - You can filter the list of orders to invoiced orders by clicking :guilabel:`Filters` and
      :guilabel:`Invoiced`.
+
+QR codes to generate invoices
+-----------------------------
+
+Customers can also request an invoice by scanning the **QR code** displayed on their receipt. Upon
+scanning, they must fill in a form with their billing information and click :guilabel:`Get my
+invoice`. By doing so, the invoice is generated and available to be downloaded. In Odoo backend, the
```

Comment:
```suggestion
invoice`. By doing so, the invoice is generated and available to be downloaded. In the backend of the POS app, the
```
Is there any a need to mention Odoo ? Do they ever leave Odoo for this feature ?